src/validation.cpp

```diff
@@ -3867,6 +3867,11 @@ bool ActivateBestChain(CValidationState &state, const CChainParams& chainparams,
                     GetMainSignals().SyncTransaction(*block.vtx[i], pair.first, i);
             }
         }
+        // Do batch verification if we reach 1 day old block,
+        BatchProofContainer* batchProofContainer = BatchProofContainer::get_instance();
+        batchProofContainer->fCollectProofs = ((GetSystemTimeInSeconds() - pindexNewTip->GetBlockTime()) > 86400) && GetBoolArg("-batching", true);
+        BatchProofContainer::get_instance()->verify();
```

Comment:
why not `batchProofContainer->verify()` ?

Reply:
ah correct, will fix, just a typo